.github/workflows/cd-common.yaml

```diff
@@ -129,8 +129,10 @@ jobs:
           chart-version: ${{ inputs.helm-chart-version }}
           cloud: ${{ matrix.cluster.cloud }}
           commit-sha: ${{ needs.deploy-env.outputs.commit-sha }}
+          doppler-token: ${{ secrets.doppler_token }}
```

Comment:
Remember to add this as a secret in the workflow definition up top

Reply:
Thanks good catch.